### PR TITLE
Fix some low-hanging fruit MSVC warnings

### PR DESCRIPTION
**Pull request by aggarw13**
Fix some low-hanging fruit MSVC warnings

Description
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c [hunk 1]
```diff
@@ -975,11 +975,11 @@ uint32_t ulIPAddress = 0uL;
 uint8_t *pucByte;
 size_t uxSourceBytesRemaining;
 uint16_t x, usDataLength, usQuestions;
-BaseType_t xDoStore = xExpected;
 #if( ipconfigUSE_LLMNR == 1 )
 	uint16_t usType = 0, usClass = 0;
 #endif
 #if( ipconfigUSE_DNS_CACHE == 1 ) || ( ipconfigDNS_USE_CALLBACKS == 1 )
+	BaseType_t xDoStore = xExpected;
```
Comment (dcgaws) [inline libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c:982]: Does this change compile regardless of the combination of these #defines set?

File: vendors/pc/boards/windows/aws_demos/config_files/FreeRTOSConfig.h [hunk 2]
```diff
@@ -135,7 +135,7 @@ extern void vLoggingPrintf( const char * pcFormat,
 #define configPRINTF( X )    vLoggingPrintf X
 
 /* Non-format version thread-safe print. */
-#define configPRINT_STRING( X )    vLoggingPrint( X )
+#define configPRINT_STRING( X )    vLoggingPrintf( X )
```
Comment (dcgaws) [inline vendors/pc/boards/windows/aws_demos/config_files/FreeRTOSConfig.h:138]: Why is this being changed back? What are the ramifications of this change?

Comment (aggarw13) [reply vendors/pc/boards/windows/aws_demos/config_files/FreeRTOSConfig.h:138]: Didn't realize that there was a non-format function available. This config lacked the declaration of the function causing an MSVC warning (of `vLoggintPrint` being undefined). 
Have added the declaration and reverted to `vLoggingPrint`

File: libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c (hunk 1 above)
Comment (aggarw13) [reply libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c:982]: I have verified that this variable is only used in the sections of the function definition guarded by `ipconfigUSE_DNS_CACHE` or `ipconfigDNS_USE_CALLBACK`
https://github.com/aws/amazon-freertos/blob/abff7f3604709c9407cf373f09986a87c75d4001/libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c#L1108-L1134

**Pull request merged**